pkg/constants/constants.go

```diff
@@ -2,9 +2,11 @@ package constants
 
 const (
 	// DefaultKubeVersion specifies a default kubernetes version.
-	DefaultKubeVersion = "v1.6.3"
+	DefaultKubeVersion = "v1.6.4"
+	// DeafultNetworkProvider specifies what CNI provider to install
+	DeafultNetworkProvider = "canal"
 	// DefaultKetoK8Image specifies the image to use for keto-k8 container
```

Comment:
Deafult is spelt Default, however you have consistently misspelt it, so it will work